docs/content/preview/troubleshoot/nodes/trouble-common.md

```diff
@@ -49,19 +51,19 @@ In these cases, the database aborts the query and returns a `40001` PostgreSQL e
 
 ## Snapshot too old: When running ysql_dump
 
-When running an `ysql_dump` command that takes too long to complete, you may get the following error:
+When running an `ysql_dump` command that takes too long to complete, you may encounter the following error:
 
 ```output
 Snapshot too old: Snapshot too old. Read point: { physical: 1628678717824559 }, earliest read time allowed: { physical: 1628679675271006 }, delta (usec): 957446447: kSnapshotTooOld
 ```
 
-When the command takes a long time to be processed, a compaction may have occurred and have deleted some rows at the snapshot the dump was started on. For large backups, we recommend using [distributed snapshots](../../../manage/backup-restore/snapshot-ysql/), which are more efficient and faster.
+When the command takes a long time to be processed, a compaction may have occurred and have deleted some rows at the snapshot the dump was started on. For large backups, it is recommended to use [distributed snapshots](../../../manage/backup-restore/snapshot-ysql/), which are more efficient and fast.
 
-If you really need to use ysql_dump, you can increase the [`--timestamp_history_retention_interval_sec`](../../../reference/configuration/yb-tserver/#timestamp-history-retention-interval-sec) gflag in yb-tserver and retry.
+If you really need to use `ysql_dump`, you can increase the [`--timestamp_history_retention_interval_sec`](../../../reference/configuration/yb-tserver/#timestamp-history-retention-interval-sec) gflag in YT-TServer and retry.
 
 ## Not able to perform operations using yb-admin after enabling encryption at transit
 
-After configuring [encryption at transit](../../../secure/tls-encryption/) for yugabyte cluster, you may get the following error when trying to use `yb-admin`:
+After configuring [encryption at transit](../../../secure/tls-encryption/) for a YugabyteDB cluster, you may encounter the following error when trying to use `yb-admin`:
 
 ```sh
 ./bin/yb-admin -master_addresses <master-addresses> list_all_masters
```

Comment:
Don't think those \n would actually be in output?

Reply:
fixed